src/states/columns.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import BaseModel
+
+from src.states.basics import ButtonState
+from src.states.chat import ChatState
+from src.states.pdf import PDFUploaderState
+
+
+class LeftColumnState(BaseModel):
+    uploader: PDFUploaderState = PDFUploaderState()
+    chat: ChatState = ChatState()
+    delete_button: ButtonState = ButtonState(name="Delete PDF")
+    clear_button: ButtonState = ButtonState(name="Clear PDF")
+
+
+class RightColumnState(BaseModel):
+    count_button: ButtonState = ButtonState(name="0/0")
```

Comment:
Maybe it would be better to have some text field instead of button here?
Cause button makes it interactive, but we don't plan to add any action to it.